install/space_checker_utils.py

```diff
@@ -0,0 +1,114 @@
+from __future__ import division
+import re
+import os
+import subprocess
+import urllib2
+
+
+def get_wget_download_size(command):
+    """Get the whole download size in bytes.
+
+    :param str command: Wget command.
+
+    :return: The actual download size in bytes.
+    :rtype: int
+    """
+    try:
+        _, args = command.split(' ', 1)
+    except IndexError:
+        return 0
+
+    download_size_command = "wget --spider --server-response {}".format(args)
+    wget = subprocess.Popen(
+            download_size_command.split(),
+            stdout=subprocess.PIPE,
+            stderr=subprocess.STDOUT)
+
+    out = wget.stdout.read()
+    match = re.search('Content-Length: (\d*)', out)
+    try:
+        # Split to keep the only the size.
+        size = int(match.group(0).split()[1])
+    except (ValueError, IndexError):
+        return 0
+    return size
+
+
+def byte_to_human(size):
+    """Transform the size in bytes into the appropriate unit of measurement.
+
+    :param int size: Size in bytes.
+
+    :return: Actual size in B/KB/MB/GB.
+    :rtype: str
+    """
+    try:
+        size = int(size)
+    except ValueError:
+        return (0, "B")
+    um = "B"
+    if size < pow(2,20):
+        size = size / pow(2,10)
+        um = "KB"
+    elif size < pow(2,30):
+        size = size / pow(2, 20)
+        um = "MB"
+    else:
+        size = size / pow(2, 30)
+        um = "GB"
+
+    return (size, um)
+
+
+def is_wget(command):
+    # Wget supports HTTP, HTTPS and FTP protocols.
+    if not command.startswith('wget '):
+        return False
+
+    # Check the url(s) is/are valid.
+    urls = re.findall(r'((?:https|http|ftp)://[^\s]+)', command)
+
+    if not urls:
+        return False
+
+    for url in urls:
+        try:
+            urllib2.urlopen(url)
```

Comment:
You are doing a urllib2.urlopen call here, so you can just use ``response.info().getheader('Content-Length')``. Why the pain of using wget?